metricbeat/module/postgresql/metricset.go

```diff
@@ -100,5 +99,10 @@ func (ms *MetricSet) Close() error {
 	if ms.db == nil {
 		return nil
 	}
-	return errors.Wrap(ms.db.Close(), "failed to close connection")
+
```

Comment:
@cmacknz special attention here

Reply:
Looks good, though probably unnecessarily decorated, but behaviour is retained.

Reply:
changed the decoration